frontend/packages/console-dynamic-plugin-sdk/src/api/types/VirtualizedGrid.ts

```diff
@@ -0,0 +1,57 @@
+import { GridCellProps } from 'react-virtualized';
+
+export type Params = {
+  index: number;
+};
+
+export type Item = {
+  [key: string]: any;
+};
+
+export type GroupedItems = {
+  [key: string]: Item[];
+};
+
+export type CellItem = string | null | Item;
+
+export type RenderHeader = (heading: string) => React.ReactNode;
+
+export type RenderCell = (item: Item) => React.ReactNode;
+
+export type GridChildrenProps = {
+  data: GridCellProps;
+  columnCount: number;
+  items: CellItem[];
+  rowCount: number;
+};
+
+export type CellSize = {
+  height: string | number;
+  width: string | number;
+};
+
+export type VirtualizedGridProps = {
```

Comment:
This type is also declared in `packages/console-shared/src/components/virtualized-grid/VirtualizedGrid.tsx`

Please update ^^ to use the `VirtualizedGridProps` type from plugin SDK package, in order to avoid duplication.

Reply:
For now we can have a copy of the type.